aux/files/js/script.js

```diff
@@ -58,4 +58,20 @@ $(document).ready(function() {
       }
     }
   });
-});
\ No newline at end of file
+});
+
+$(function(){
+
+  function toggleRow(){
+    if ($(this).hasClass('success') || $(this).hasClass('danger')){
+      var title = $(this).attr('title');
+      var val = title.replace(/[ \/]/g, '');  
+      addData(this, val);
+    } else if ($(this).hasClass('plot_btn')){
+      addData(this, 'all');
+    }
+  }
+
+  $("td, .plot_btn").click(toggleRow);
```

Comment:
This binds the same event handler 2500 times, once per row. Pretty wasteful, don't you think? It's preferred to use event delegation in such cases - http://learn.jquery.com/events/event-delegation/